style.css

```diff
@@ -0,0 +1,180 @@
+* {
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+#mainContainer {
+  background-color: white;
+  width: 80%;
+  height: 100%;
+  margin: 0px auto;
+  padding: 0px 20px;
+}
+#root {
+  margin: 10px auto;
+  width: 100%;
+  height: 460px;
+  display: flex;
+  flex-wrap: wrap;
+  border: 0px solid rgb(214, 27, 231);
+}
+
+#headerDiv {
+  margin: 6px auto;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  background-color: burlywood;
+}
+#newGridBut {
+  padding: 6px 23px;
+  margin: 0px;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+
+#clearBut {
+  padding: 6px;
+  margin: 0px;
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+.divElement {
+  border: 1px solid rgb(245, 245, 167);
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+}
+
+.hoverClass {
+  background-color: darkcyan;
+}
+
+.radioBtns {
+  width: 220px;
+  height: 80px;
+}
+#BtnsDiv {
+  padding-top: 14px;
+  margin-left: 6px;
+  height: 100px;
+  width: 120px;
+}
+.radioBtns input {
+  margin-top: 0px;
+}
+
+#clrSelected {
+  width: 12px;
+  height: 12px;
+  border: 1px solid rgb(119, 115, 115);
+  display: inline-block;
+}
+
+input.chk-btn {
+  display: none;
+}
+
+#clearBut {
+  padding: 6px;
+  margin-top: 4px;
+}
+
+input.chk-btn + label {
+  border: 1px solid rgb(59, 59, 59);
+  background: #efefef;
+  padding: 5px;
+  margin-top: 4px;
+  display: inline-block;
+  font-size: 15px;
+}
+/*input.chk-btn:not(:checked) + label:hover {
+  box-shadow: 0px 1px 3px;
+}*/
```

Comment:
for a clean code, any unused code or such comment has to be removed

Reply:
ok i removed it